Makefile/Makefile.sublime-syntax

```diff
@@ -12,6 +12,7 @@ file_extensions:
   - OCamlMakefile
   - mak
   - mk
+first_line_match: '^#!/usr/bin/(make|env\s+make)'
```

Comment:
This should probably be something more like: `'^#!/usr/bin/(make|env\s+make)\b'` to ensure `make` isn't part of a longer word.